src/test/groovy/integration/TestScalaMultiVersionPluginIntegration.groovy

```diff
@@ -69,6 +69,20 @@ class TestScalaMultiVersionPlugin extends GroovyTestCase implements SimpleProjec
         assert result.output.contains("t2 2.2.2")
     }
 
+    void testExcludeTaskPassesThrough() {
+        def result = GradleRunner.create()
+            .withProjectDir(projectDir)
+            .withArguments("t1", "t2", "--exclude-task=t2", "-PscalaVersions=1.1.1,2.2.2")
+            .build()
+        assert result.taskPaths(SUCCESS).toSorted() == [":t1",
+                                                        ":recurseWithScalaVersion_2.2.2",
+                                                        ":codeProject:t1"].sort()
```

Comment:
prior to the change, `:t2` was absent as expected, but `:codeProject:t2` was present because the task exclusions were overwritten by the plugin when recursing.